### ISSUE TITLE
Commit notification?

### ISSUE THREAD

**Issue by eudoxos** [oliverguenther/openproject-revisions_git#14]
This is a non-critical request for enhancement than an issue. Could the plugin notify all project memebers (or only watchers, with choice like "work package" has) of commits in the repo?

**Comment by oliverguenther** [comment]
I don't think this belongs in the core functionality of this plugin, as its details are highly subjective and would require a significant amount of customization (e.g., which roles to mail). This effort increases when using the custom watcher feature that currently exist for work packages.

However, I would like to re-initiate the hooks part of the original plugin, which allows [adding Git Hooks through Gitolite](http://gitolite.com/gitolite/cookbook.html#cb-hooks).

The added hook could then easily call a custom plugin with a rake task to mail all users. I think that there would be many desirable use-cases for these kinds of hooks.
